### ISSUE TITLE
Feature Request: Making specific entries not eligible for HIBP checks

### ISSUE THREAD

**Issue by jeff9315** [mihaifm/HIBPOfflineCheck#36]
I've got a number of Keepass entries in which the passwords contain phone numbers (library password requirement), PINs, the last 4 digits of SSN, etc. Since these are short numeric fields, HIBP always shows them as breached. I wonder if it would make sense to have a way to exempt specific entries from being checked. 

The only way I can think of would be to have a separate field (e.g., "Skip HIBP Check") that the user could check off. But this would require an additional UI feature that may not be worthwhile for you to implement.

Or the user (me) could move all these PINs, etc out of the Password field as a work-around.

Thoughts?

**Comment by mihaifm** [comment]
You can clear the HIBP status with the right click option, so the status remains blank for those entries. The only downside is they will get rechecked if you ever perform "Check All Passwords". But I guess you only need to check all passwords the first time you use the plugin.

Not sure how the GUI part you are suggesting should work. Are you saying there should be another column with the additional check box options? That might be too much of a clutter. 
I'm thinking a right click option might work better, like "Exclude from HIBP" which will set the HIBP status to "N/A".

**Comment by homes32** [comment]
> I'm thinking a right click option might work better, like "Exclude from HIBP" which will set the HIBP status to "N/A".

To expand further on this HIBP could check the status before updating it or checking the password and ignore the entry if the status contains "N/A". Then you don't need a separate field.

**Comment by mihaifm** [comment]
Yea, that was my thought as well. It's not hard to do, just need to find some time for it.

**Comment by jeff9315** [comment]
Yes, I agree the Right-click option would be best. And I also like the idea of just using the Status Column to do double-duty. "N/A" could signify both that it should NOT be checked and that it was NOT checked.

Thanks for considering this!

Jeff

**Comment by jeff9315** [comment]
Perfect! Thanks!!
